testcafe/basic/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM testcafe/testcafe
```

(optional) Would personally prefer a tag here. Just easier to figure out what TestCafe version was used without digging through logs 🔍 
```suggestion
FROM testcafe/testcafe:3.5.0
```